mungegithub/mungers/approvers/approvers_test.go

```diff
@@ -656,3 +663,82 @@ You can cancel your approval by writing ` + "`/approve cancel`" + ` in a comment
 		t.Errorf("GetMessage() = %+v, want = %+v", *got, want)
 	}
 }
+
+func TestGetMessageApprovedIssueAssociated(t *testing.T) {
+	ap := NewApprovers(
+		Owners{
+			filenames: []string{"a/a.go", "b/b.go"},
+			repo: createFakeRepo(map[string]sets.String{
+				"a": sets.NewString("Alice"),
+				"b": sets.NewString("Bill"),
+			}),
+		},
+	)
+	ap.AssociatedIssue = 12345
+	ap.AddAuthorSelfApprover("John", "REFERENCE")
+	ap.AddApprover("Bill", "REFERENCE", false)
+	ap.AddApprover("Alice", "REFERENCE", false)
+
+	want := `[APPROVALNOTIFIER] This PR is **APPROVED**
```

Comment:
Yeah, let's try to keep this as simple as possible. We don't have the author information in this part of the code.

Let's assume people are not going to game that that much, plus you still need to be the OWNER of at least one of the modified files in order to add the `no-issue` bit.